packages/apify/src/actor.ts

```diff
@@ -1606,6 +1611,73 @@ export interface ApifyEnv {
      * in megabytes (ACTOR_MEMORY_MBYTES)
      */
     memoryMbytes: number | null;
+
+    /**
+     * If set to "1", the web browsers inside the Actor should run in headless
+     * mode because there is no windowing system available. (APIFY_HEADLESS)
+     */
+    headless: string | null;
```

Comment:
Why aren't these booleans? We already parse Dates above so this object already has parsed env, it is not 1:1 raw strings